engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/number/decimal/TruncateNode.java

```diff
@@ -0,0 +1,31 @@
+package org.enso.interpreter.node.expression.builtin.number.decimal;
+
+import com.oracle.truffle.api.CompilerDirectives;
+import com.oracle.truffle.api.nodes.Node;
+import com.oracle.truffle.api.profiles.ConditionProfile;
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.node.expression.builtin.number.utils.BigIntegerOps;
+import org.enso.interpreter.runtime.number.EnsoBigInteger;
+
+@BuiltinMethod(
+    type = "Decimal",
+    name = "truncate",
```

Comment:
Shouldn't `truncate_builtin` name be used instead? I thought we have such convention. At least I use that convention. Feel free to ignore.

Reply:
Done